### DISCUSSION TITLE
Quality Issue In Larger CBZs

### DISCUSSION THREAD

**Discussion by PvtUnderdog** [devnoname120/epubviewer#14] [General] [answered]
I just installed this on a Nextcloud 26 server, then switched to my non-admin user to open a CBZ. I used the highest quality file I have, so I was surprised to see that it looked quite bad. When I zoomed in all the way, it was as clear as it gets, but zoom to fit was illegible. Naturally, it was even worse on the smaller screen of my phone. As a sanity check, I opened the same file in a couple different local readers and it's fully legible on my phone, even in two-page mode.

I can't claim to know the inner workings, but maybe the *down*scaling algorithm isn't applying?

P.S. This is my first interaction on GitHub, so please forgive me if this is the wrong spot/formatting.

**Comment by devnoname120** [comment, marked as answer]
@PvtUnderdog Fixed in https://github.com/devnoname120/epubviewer/releases/tag/1.9.6
